addons/point_of_sale/models/pos_config.py

```diff
@@ -690,8 +689,6 @@ def _get_available_product_domain(self):
         ]
         if self.limit_categories and self.iface_available_categ_ids:
             domain.append(('pos_categ_ids', 'in', self._get_available_categories().ids))
-        if self.iface_tipproduct:
-            domain = OR([domain, [('id', '=', self.tip_product_id.id)]])
```

Comment:
I didn't see that `tip product` included. We should include it in the result of `_get_special_products`.

Reply:
I think its added [here](https://github.com/odoo/odoo/pull/175796/files#diff-4c6e412c7d8f4df2a05831547e7df93d0b91f510d03b7b3ed0d689a18f5dae44R812-R813).
Also there is an [enterprise PR](https://github.com/odoo/enterprise/pull/69347) as well i'm sorry i forgot to kmention in the description now added it.